### PR TITLE
loading

### PR DESCRIPTION
**Pull request by DNaunin**
Instead of just writing "Loading", there is now a loading screen with Lottie File, while a user waits that his file is uploaded.

### CODE REVIEW

**Review by cboenke — APPROVED** [review]
LGTM 🚀

File: pages/upload.tsx [hunk 1]
```diff
@@ -46,14 +47,9 @@ export default function Upload() {
...
           />
 
-          {loading ? (
-            <div className={styles.subtitle}>Loading... </div>
-          ) : (
-            <img src={image} className={styles.image} />
-          )}
+          {loading ? <Loading /> : <img src={image} className={styles.image} />}
         </form>
         <Pagelink href="/design">Go to Design Page</Pagelink>
```
Comment (RonjaMeinecke) [inline pages/upload.tsx:52]:
```suggestion
        <PageLink href="/design">Go to Design Page</Pagelink>
```

Comment (RonjaMeinecke) [inline pages/upload.tsx:52]:
```suggestion
        <Pagelink href="/design">Go to Design Page</PageLink>
```

File: pages/upload.tsx [hunk 2]
```diff
@@ -2,6 +2,7 @@ import styles from "../styles/Upload.module.css";
 import Footer from "../components/footer/Footer";
 import React, { useState } from "react";
 import Pagelink from "../components/pagelink/Pagelink";
```
Comment (RonjaMeinecke) [inline pages/upload.tsx:4]:
```suggestion
import PageLink from "../components/pagelink/Pagelink";
```

**Review by RonjaMeinecke — APPROVED** [review]
🦾GJ

**Review by SilviaBarcellini — APPROVED** [review]
All very nice 🚀

**Review by JuKahl — APPROVED** [review]
GJ 🧚‍♀️

**Pull request merged**